### DISCUSSION TITLE
Is there any way to disable logs ?

### DISCUSSION THREAD

**Discussion by agrezo** [withcatai/node-llama-cpp#83] [Q&A] [answered]
Hello guys ✌🏻
It's maybe a dumb question but is it possible to disable all the logs in console ? Because i can't find this in documentation.

<img width="987" alt="Screenshot 2023-11-01 at 12 42 33" src="https://github.com/withcatai/node-llama-cpp/assets/8040580/72a24583-2d01-4172-8201-f08dded8a9ba">

Thank you!

**Comment by giladgd** [comment, marked as answer]
@agrezo AFAIK, there's no way to disable `llama.cpp`'s logs.
You can open an [issue on `llama.cpp`'s repo](https://github.com/ggerganov/llama.cpp/issues) and I'll then add support for it in `node-llama-cpp`

**Comment by OlleMattsson** [comment]
I'm wondering about the same thing.

In this discussion over att the llama.cpp repo, it is suggested to pipe stderr to /dev/null
https://github.com/ggerganov/llama.cpp/discussions/1758

But... no clue how to implement that in node-llama-cpp, yet.
